services/vpc.go

```diff
@@ -66,6 +67,21 @@ func (c *client) FindVPC(vpcName string) (string, error) {
 	return vpcList[0].ID, nil
 }
 
+// DeleteSecurityGroupViaVPC deletes security group from give VPC Id
```

Comment:
`deletes security group from give VPC Id` should be smth. like `deletes all security groups of VPC`